scripts/prepare_release.sh

```diff
@@ -85,14 +85,57 @@ if [[ ! -e "apikey.txt" ]]; then
     exit 1
 fi
 
+
+###################
+#  VALIDATE REPO  #
+###################
+
+# Ensure the checked out branch is master
+CHECKED_OUT_BRANCH="$(git branch | grep "*" | awk -F ' ' '{print $2}')"
+if [[ $CHECKED_OUT_BRANCH != "master" ]]; then
+    read -p "[WARN] You are on the '${CHECKED_OUT_BRANCH}' branch, not 'master'. Continue? (Y/n) " CONTINUE
+    echo
+
+    if ! [[ $CONTINUE == "Y" ]]; then
+        echo "[INFO] You chose not to continue."
+        exit 1
+    fi
+fi
+
+# Ensure the branch does not have local changes
+if [[ `git status --porcelain` ]]; then
```

Comment:
earlier we used $(..) syntax, maybe replace the backticks for $(...)

Reply:
Done